src/autoscaling/system_status.js

```diff
@@ -213,8 +213,8 @@ class SystemStatus {
         const wAvg = weightedAvg(values, weights);
         return {
             isOverloaded: wAvg > ratio,
-            maxOverloadedRatio: ratio,
-            actualRatio: wAvg,
+            limitRatio: ratio,
+            actualRatio: Math.round(wAvg * 1000) / 1000,
```

Comment:
What's the point of the `Math.round`?

Reply:
It's just that the log looks really bad:

```
INFO: AutoscaledPool state {"currentConcurrency":3,"desiredConcurrency":1,"systemStatus":{"isSystemIdle":false,"memInfo":{"isOverloaded":false,"maxOverloadedRatio":0.2,"actualRatio":0},"eventLoopInfo":{"isOverloaded":false,"maxOverloadedRatio":0.4,"actualRatio":0.01992771979599419},"cpuInfo":{"isOverloaded":true,"maxOverloadedRatio":0.4,"actualRatio":0.43579545454545454},"clientInfo":{"isOverloaded":false,"maxOverloadedRatio":0.3,"actualRatio":0}}}
```

There's no need to have 17 digits for the ratio, 3 is enough (IMHO). I assume these values are only used in log, right?

Reply:
Oh, so the point of the whole `* 1000) / 1000` is rounding to precision of 3. Ok.